lib/registries/addon/discover/styles.scss

```diff
@@ -0,0 +1,35 @@
+.ShareLogo {
+    background-size: contain !important;
+    background-repeat: no-repeat !important;
+    background: url('img/share-logo-white.png');
+    display: inline-block;
+    height: 33px;
+    vertical-align: middle;
+    width: 69px;
+}
+
+.SortDropDown__List {
+    background-color: $color-bg-gray-light;
+}
+
+.SortDropDown__Option {
+    text-align: left;
+    background-color: transparent;
+    cursor: pointer;
+    color: #000;
+    text-decoration: none;
+    width: 100%;
+}
+
+.ActiveFilters--RegistrationType {
+    background-color: #d7e6e9;
```

Comment:
Could you put all colors in `app/styles/_variables`? It helps us reuse colors rather than adding a million slightly different shades, and better positions us to adopt an actual color palette.